suite-common/message-system/config/config.v1.json

```diff
@@ -29,7 +29,9 @@
                     "hu": "A Coinjoin 2024. június 1-jén megszűnt a Trezor Suite-ban. Az Ön pénzeszközei továbbra is elérhetőek maradnak, és nincs szükség semmilyen intézkedésre.",
```

Comment:
<img width="606" alt="Screenshot 2024-08-15 at 22 08 39" src="https://github.com/user-attachments/assets/fdc68093-d1e7-4b69-bcb3-8b104e3946af">
please add official langs here, remove community, thxs

Reply:
Resolved in 5987a6a1b02e0d2581c8a6587c8a49d68c2219c1